### DISCUSSION TITLE
Parse and warn for invalid `sizes` prop on Image component

### DISCUSSION THREAD

**Discussion by styfle** [vercel/next.js#64640] [Ideas]
As a follow up to https://github.com/vercel/next.js/pull/31064#issuecomment-964022993 we can use the description from the [html spec](https://html.spec.whatwg.org/multipage/images.html#parsing-a-sizes-attribute) to parse the `sizes` prop and warn if its invalid.

We probably don't want to try evaluating the media condition, just parsing the string should be sufficient.

**Comment by qjoner** [comment]
- name: Cancel Workflow Action
  uses: styfle/cancel-workflow-action@0.9.1
